### PR TITLE
Update MBC 2 Flux

### PR DESCRIPTION
**Pull request by VoXaN24**

### CODE REVIEW

**Review by freearhey — APPROVED** [review]

**Review by its0x08 — APPROVED** [review]

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
gets 403'd over some time + Xtream-Codes server.
image

File: streams/ae.m3u [hunk 1]
```diff
@@ -71,8 +71,8 @@ https://shls-mbc1ksa-prod-dub.shahid.net/out/v1/451b666db1fb41c7a4bbecf7b4865107
 https://shls-mbc1-usa-prod.shahid.net/out/v1/1b559e832c3f40f996c1984245b3b24b/index.m3u8
 #EXTINF:-1 tvg-id="MBC1USA.ae" status="online",MBC 1 North America (1080p)
 https://shls-mbc1na-prod-dub.shahid.net/out/v1/84ab37e99d6e4b16b33c6600f94f6ccb/index.m3u8
-#EXTINF:-1 tvg-id="MBC2.ae" status="blocked",MBC 2
-http://zinachannels.zinamedia.net/ar-s/mbc-2/index.m3u8
+#EXTINF:-1 tvg-id="MBC2.ae" status="blocked",MBC 2 (720p)
+http://multies.xyz:8000/live/22-5-MbC2-Mx-AcT/G8bwyFcnsM/106.m3u8
```
[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply streams/ae.m3u:75]: The zina link 403s on my side.

[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply streams/ae.m3u:75]: If you talked about the second link, it's from an Xtream-Codes server. Those are quite sketchy in terms of how they work, hence why I didn't accept this PR.

**Review by ghost — APPROVED** [review]

**Review by BellezaEmporium — COMMENTED** [review]
Quite reticent in the idea of adding an Xtream-Codes server in our playlists.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**